card-games-vite/src/utils/deckOfCardsApi.js

```diff
@@ -0,0 +1,84 @@
+import { checkResponse } from "./api";
+
+const deckOfCardsUrl = "https://deckofcardsapi.com";
+
+function createNewDeck(deckNumber) {
+  return fetch(
+    `https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=${deckNumber}`
+  ).then(checkResponse);
+}
+
+function drawCard(deck_id, numberOfCards) {
+  return fetch(
+    `https://deckofcardsapi.com/api/deck/${deck_id}/draw/?count=${numberOfCards}`
```

Comment:
`https://deckofcardsapi.com/api/deck` is repeated 11 times, so you need to define `baseUrl` for it